wp_api_integration_tests/src/lib.rs

```diff
@@ -14,13 +14,19 @@ include!(concat!(env!("OUT_DIR"), "/generated_test_credentials.rs"));
 
 #[derive(Debug, Default)]
 pub struct TestCredentials {
-    pub site_url: String,
-    pub admin_username: String,
-    pub admin_password: String,
-    pub admin_password_uuid: String,
-    pub subscriber_username: String,
-    pub subscriber_password: String,
-    pub subscriber_password_uuid: String,
+    pub site_url: &'static str,
+    pub admin_username: &'static str,
+    pub admin_password: &'static str,
+    pub admin_password_uuid: &'static str,
+    pub subscriber_username: &'static str,
+    pub subscriber_password: &'static str,
+    pub subscriber_password_uuid: &'static str,
+    pub author_username: &'static str,
+    pub author_password: &'static str,
+    pub password_protected_post_id: i32,
+    pub password_protected_post_password: &'static str,
+    pub password_protected_post_title: &'static str,
+    pub trashed_post_id: i32,
 }
```

Comment:
This seems more performant, but less ergonomic since we have to do `.to_string()` all the time – WDYT?

Reply:
This change is due to the addition of a `i32` field, because I can no longer do the `.to_string()` part in `build.rs`. It may be possible to still use `String` if we feel strongly about avoiding `to_string()` everywhere. Let me know if you feel that way and I'll look into it.

Reply:
Don't feel strongly about it 🤷